src/main/java/org/gbif/api/vocabulary/License.java

```diff
@@ -0,0 +1,84 @@
+package org.gbif.api.vocabulary;
+
+import com.google.common.base.Strings;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Enumeration of the set of licenses GBIF supports for applying to a dataset. The license provides a standardised way
+ * to define appropriate uses of a dataset.
+ * </br>
+ * GBIF's recommended best practice is to use the most recent license version, which for CC-BY and CC-BY-NC is 4.0.
+ * This is in line with the recommendation from Creative Commons.
+ *
+ * @see <a href="https://creativecommons.org/faq/#why-should-i-use-the-latest-version-of-the-creative-commons-licenses">Creative Commons recommendation</a>
+ * @see <a href="http://www.gbif.org/terms/licences">GBIF Licensing</a>
+ */
+public enum License {
+
+  /**
+   * Creative Commons Zero / Public Domain version 1.0. Technically a waiver, not a license.
+   *
+   * @see <a href="http://creativecommons.org/publicdomain/zero/1.0/legalcode">legal document</a>
+   */
+  CC0_1_0("Public Domain (CC0 1.0)", "http://creativecommons.org/publicdomain/zero/1.0/legalcode"),
+  /**
+   * Creative Commons Attribution version 4.0.
+   *
+   * @see <a href="http://creativecommons.org/licenses/by/4.0/legalcode">legal document</a>
+   */
+  CC_BY_4_0("Creative Commons Attribution (CC-BY) 4.0", "http://creativecommons.org/licenses/by/4.0/legalcode"),
```

Comment:
Please refer to the pull request description above for an explanation why versions should be supported in my opinion. Thanks.